include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -259,6 +260,19 @@ typedef enum OrtMemType {
   OrtMemTypeDefault = 0,                // the default allocator for execution provider
 } OrtMemType;
 
+typedef enum OrtCudnnConvAlgoSearch {
+  EXHAUSTIVE,   // expensive exhaustive benchmarking using cudnnFindConvolutionForwardAlgorithmEx
+  HEURISTIC,    // lightweight heuristic based search using cudnnGetConvolutionForwardAlgorithm_v7
+  DEFAULT,      // default algorithm using CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_PRECOMP_GEMM
+} OrtCudnnConvAlgoSearch;
+
+typedef struct OrtCUDAProviderOptions {
+  int device_id;  // cuda device with id=0 as default device.
+  OrtCudnnConvAlgoSearch cudnn_conv_algo_search;  // cudnn conv algo search option
+  size_t cuda_mem_limit;   //  default cuda memory limitation to maximum finite value of size_t.
+  int arena_extend_strategy;    // default area extend strategy to KNextPowerOfTwo.
+} OrtCUDAProviderOptions;
+
```

Comment:
int device_id;  // Cuda device (Use 0 for default)
  OrtCudnnConvAlgoSearch cudnn_conv_algo_search;  // (is there a recommended one?)
  size_t cuda_mem_limit;   //  Cuda memory limit (Use 0 for no limit)
  int arena_extend_strategy;    // KNextPowerOfTwo is recommended, see (enum name) for more info

Reply:
Just some phrasing and changes to make it clearer what a user of the API should do.

Also no real need to comment when the comment matches the variable name :)